packages/web-components/package.json

```diff
@@ -17,7 +17,7 @@
     "scripts": {
         "build.stencil": "stencil build --docs",
         "build.storybook": "build-storybook -s ./dist",
-        "build": "npm run build.stencil && npm run build.storybook",
+        "build": "npm run build.stencil",
```

Comment:
just curious why was this needed? can we remove build.stencil then?

Reply:
I think it's because build.tests in the root package.json does lerna build, but we don't need storybook to build to run the tests. I think we could change the root command to be 
`lerna run --scope @astrouxds/web-components build.stencil && lerna run --scope @astrouxds/react build`
if we wanted to keep the web-components build command as it was

Reply:
Makes sense! And I prefer it this way 100%. 9/10 times I only want to run stencil so this saves me from typing 8 characters